locale/ja/LC_MESSAGES/django.po

```diff
@@ -1381,14 +1254,8 @@ msgid "We are not running any studies at this time!"
 msgstr "今のところ勉強はしていません!"
 
 #: web/templates/web/studies-list.html:93
-msgid ""
-"Looking for more ways to contribute to research from home? Check out <a href="
-"\"https://childrenhelpingscience.com/\" target=\"_blank\" rel=\"noreferrer "
-"noopener\">Children Helping Science</a> for even more studies!"
-msgstr ""
-"自宅で研究に貢献する方法をお探しですか?<a href=\"https://"
-"childrenhelpingscience.com/\" target=\"_blank\" rel=\"noreferrer noopener"
-"\">Children Helping Science</a>では、さらに多くの研究をご覧いただけます。"
+msgid "Looking for more ways to contribute to research from home? Check out <a href=\"https://childrenhelpingscience.com/\" target=\"_blank\" rel=\"noreferrer noopener\">Children Helping Science</a> for even more studies!"
+msgstr "自宅で研究に貢献する方法をお探しですか?<a href=\"https://childrenhelpingscience.com/\" target=\"_blank\" rel=\"noreferrer noopener\">Children Helping Science</a>では、さらに多くの研究をご覧いただけます。"
```

Comment:
This might be something on our end, but should we excluding HTML from translations?